hack/update-staging-godeps.sh

```diff
@@ -71,7 +71,9 @@ function updateGodepManifest() {
     go run "${KUBE_ROOT}/staging/godeps-json-updater.go" --godeps-file="${TMP_GOPATH}/src/k8s.io/${repo}/Godeps/Godeps.json" --client-go-import-path="k8s.io/${repo}"
 
     # commit so that following repos do not see this repo as dirty
-    git add vendor >/dev/null
+    # pipe through true so that command script won't error if there are no
+    # files to add
+    git add vendor/ >/dev/null || true
```

Comment:
I'm running this against a clean copy of `master`. If I add in a debugging line to print my current working directory right before that command, it's failing at:
```
Adding vendor folder in /var/folders/lw/b5k2lvx53j98z_c0z8cyk_bcgjcp8_/T/tmp.VdZc7RBU/go/src/k8s.io/apimachinery
fatal: pathspec 'vendor' did not match any files
```